assets/multiqc_config.yml

```
@@ -1,5 +1,7 @@
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/differentialabundance/tree/dev" target="_blank">nf-core/differentialabundance</a> analysis pipeline. For information about how to interpret these results, please see the <a href="https://nf-co.re/differentialabundance/dev/docs/output" target="_blank">documentation</a>.
+  This report has been generated by the <a href="https://github.com/nf-core/differentialabundance/tree/dev" target="_blank">nf-core/differentialabundance</a>
+  analysis pipeline. For information about how to interpret these results, please see the
+  <a href="https://nf-co.re/differentialabundance/dev/docs/output" target="_blank">documentation</a>.
```

```suggestion
   This report has been generated by the <a href="https://github.com/nf-core/differentialabundance/tree/dev" target="_blank">nf-core/differentialabundance</a> analysis pipeline. For information about how to interpret these results, please see the <a href="https://nf-co.re/differentialabundance/dev/docs/output" target="_blank">documentation</a>.
```

Not sure why that was split over lines

---

No idea either, I found it weird as well but wanted to stick to the template update